model/common/tests/grid_tests/utils.py

```diff
@@ -97,15 +99,19 @@ def valid_boundary_zones_for_dim(dim: dims.Dimension):
     yield from _domain(dim, zones)
 
 
-@functools.cache
-def run_grid_manager(experiment_name: str, num_levels=65, transformation=None) -> gm.GridManager:
+def run_grid_manager(
+    experiment_name: str,
+    backend: gtx_backend.Backend = gtx.gtfn_cpu,
+    num_levels=65,
```

Comment:
is it ok to hard code this?

Reply:
I set it as default. It is the same default as before.